scripts/e2e_test_tosa.sh

```diff
@@ -58,14 +56,14 @@ echo "Optimizing tf dialect"
 python optimize_tf_dialect.py "$OUTPUT_DIR"/model_tf.mlir "$OUTPUT_DIR"/model_tf_opt.mlir
 
 echo "Converting tf dialect to tosa dialect"
-"$TF_OPT" --tf-to-tosa-pipeline "$OUTPUT_DIR"/model_tf_opt.mlir > "$OUTPUT_DIR"/model_tosa.mlir
+python tf_to_tosa_dialect.py "$OUTPUT_DIR"/model_tf_opt.mlir "$OUTPUT_DIR"/model_tosa.mlir
 
 echo "Removing tf._input_shapes attribute"
 sed "s/tf._input_shapes =.*]//" "$OUTPUT_DIR"/model_tosa.mlir > "$OUTPUT_DIR"/model_tosa_noattr.mlir
 sed -i "s/, }/}/" "$OUTPUT_DIR"/model_tosa_noattr.mlir
 
 echo "Fixing function name"
-FUNCTION_NAME=$(grep -oe "@[^(]*" "$OUTPUT_DIR"/model_tosa_noattr.mlir)
+FUNCTION_NAME=$(grep -m 1 -oe "@[^(]*" "$OUTPUT_DIR"/model_tosa_noattr.mlir)
```

Comment:
This change is needed, because it breaks the next sed command if grep returns multiple lines. I think grep only returns multiple lines with the tosa translation, but it certainly would make the other scripts more robust.

Reply:
In that case please send a followup.